src/components/ClusterNodes/ClusterTerminal.tsx

```diff
@@ -148,7 +168,10 @@ export default function ClusterTerminal({
                         setSocketConnection(SocketConnectionType.DISCONNECTED)
                     })
             }
-            toggleOptionChange()
+            
```

Comment:
This is supposed to be in useEffect's clean up function. right? Please move it out from try catch block

Reply:
no its not